packages/gitbook/src/components/SpaceLayout/SpaceLayout.tsx

```diff
@@ -95,6 +94,57 @@ export function SpaceLayoutServerContext(props: SpaceLayoutProps) {
     );
 }
 
+function getSpaceVariants(context: GitBookSiteContext) {
```

Comment:
```suggestion
function categorizeVariants(context: GitBookSiteContext) {
```
To avoid confusion because `getSpaceVariants` hints something about a specific space.